src/Raven.Server/Documents/Replication/IncomingReplicationHandler.cs

```diff
@@ -1418,6 +1418,18 @@ protected override long ExecuteCmd(DocumentsOperationContext context)
                                             _replicationInfo.Logger.Info(
                                                 $"Conflict check resolved to Conflict operation, resolving conflict for doc = {item.Id}, with change vector = {item.ChangeVector}");
 
+                                        if (hasLocalClusterTx == hasRemoteClusterTx)
```

Comment:
Can this be that both are false here?

Reply:
yes, both false is the common case, normal conflict